modules/osdk-monitoring-prometheus-servicemonitor.adoc

```diff
@@ -3,18 +3,12 @@
 // * operators/operator_sdk/osdk-monitoring-prometheus.adoc
 
 [id="osdk-monitoring-prometheus-servicemonitor_{context}"]
-= ServiceMonitor resources
+= Service monitors
 
-A ServiceMonitor is a Custom Resource Definition (CRD) provided by the
-Prometheus Operator that discovers the `Endpoints` in Service objects and
-configures Prometheus to monitor those pods.
+A `ServiceMonitor` is a custom resource definition (CRD) provided by the Prometheus Operator that discovers the `Endpoints` in `Service` objects and configures Prometheus to monitor those pods.
```

Comment:
Wouldn't this be a CR, not a CRD, and a user then creates `ServiceMonitor` objects?